yaml/service.go

```diff
@@ -61,6 +64,32 @@ func (s *ServiceSlice) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return err
 	}
 
+	// iterate through each element in the service slice
+	for _, service := range *serviceSlice {
+		// implicitly set `pull` field if empty
+		if len(service.Pull) == 0 {
+			service.Pull = constants.PullNotPresent
+		}
+
+		// TODO: remove this in a future release
+		//
+		// handle true deprecated pull policy
+		//
+		// a `true` pull policy equates to `always`
+		if strings.EqualFold(service.Pull, "true") {
+			service.Pull = constants.PullAlways
+		}
+
+		// TODO: remove this in a future release
+		//
+		// handle false deprecated pull policy
+		//
+		// a `false` pull policy equates to `not_present`
+		if strings.EqualFold(service.Pull, "false") {
+			service.Pull = constants.PullNotPresent
+		}
```

Comment:
Set the deprecated `false` pull policy to `not_present` for the `github.com/go-vela/types/yaml.Service` type